### ISSUE TITLE
We can't edit the ontology in parallel

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4180]
cc: @nicolevasilevsky @matentzn I post this question in the Slack channel, but I also tag you here.

We have a question in PomBase regarding collaborative editing of our phenotype ontology.

We use the file `src/ontology/fypo-edit.obo` in [FYPO repository](https://github.com/pombase/fypo) to make our edits, and we have a release pipeline after that.

We would like to be able to work on branches on `fypo-edit` in parallel, and merge changes for a certain release. The problem is the line that starts with `-owl-axioms:` (with a lot of `\n` characters). Any time the file is saved (even without changes, e.g. add a subclass, undo, save), this line changes. Since all the information is in a single line, there is no way to merge different contributions prior to the release. Is there a way to tell protege to store this information in a different way so that we can edit the ontology in parallel?

This would also be important for revising changes prior to making a release, since right now we cannot see changes to logical definitions using `git diff`.

**Comment by matentzn** [comment]
What tool are you using to edit fypo? Protege or is anyone still using obo edit?

**Comment by manulera** [comment]
We are using protege

**Comment by matentzn** [comment]
Excellent. Can I confirm that all of you are using Protege, and no one will use OBO Edit again? If so, the problem is easily solved.

**Comment by manulera** [comment]
Yes, indeed only me and @ValWood are doing this now, and we both use Progete 5.5

**Comment by matentzn** [comment]
Excellent. Solution:

IN the terminal run the following command:

```
cd src/ontology
sh run.sh robot convert -i fypo-edit.obo -o fypo-edit.ofn
mv fypo-edit.ofn fypo-edit.owl
rm fypo-edit.owl
```

From now on, you will use OWL functional syntax as the editing language like all other phenotype ontologies and you wont run into merge conflicts.

**Comment by manulera** [comment]
Thanks for the quick reply @matentzn.

I will wait for an OK on this from @ValWood and @kimrutherford before doing it, and then I will close the issue.

**Comment by ValWood** [comment]
Sounds good. I won't be using obo-edit

**Comment by manulera** [comment]
Hi @matentzn, I was about to do what you said here:

> IN the terminal run the following command:

I wonder if in the last command, you mean `rm fypo-edit.obo`

**Comment by matentzn** [comment]
Oops yes. `rm fypo-edit.obo` Sorry about that!

**Comment by manulera** [comment]
Solved in #4192 documentation in the wiki changed, and this extra step added:

https://github.com/pombase/fypo/wiki/FYPO-release-procedure

See `% sh run.sh make normalise_xsd_string`
